mitodl/open-discussions
# REST API for channels

#### What are the relevant tickets?
Part of #29 

#### What's this PR do?
Implements rest API for channels

#### How should this be manually tested?
You can go to `/api/v0/channels/` and play around with it using the DRF HTML view. You should also be able to go to `/api/v0/channels/<subreddit>/` to view a particular channel or to patch it
